app/models/wpcc/percent.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  class Percent
+    def self.choose_highest(period_percent, user_percent)
```

Comment:
Technically all we're doing is comparing 2 percents. My original thought was PercentUtility